client/dashboard/sites/overview/trend-comparizon-badge.tsx

```diff
@@ -42,7 +42,7 @@ export default function TrendComparisonBadge( {
 					size={ 16 }
 					fill="currentColor"
 					icon={ negative ? arrowDown : arrowUp }
-					className="components-badge__icon"
+					className="site-overview-card__badge-icon"
```

Comment:
We should probably add a TODO here to refactor away from a custom icon in the `children` when `Badge` will updated to support custom icons  ( DS-203 )

Reply:
Done in dc66400f1e80e7845ba94beaf940f887f90a62f6